packages/support/resources/views/components/input/wrapper.blade.php

```diff
@@ -174,6 +175,26 @@
                 />
             @endif
 
+            @if ($isRevealable)
+                <div @class([
+                    $actionsClasses,
+                    'select-none'
+                ]) >
+                    <x-filament::icon-button
+                        icon-alias="forms::components.text-input.show-password"
+                        icon="heroicon-o-eye"
+                        color="gray"
+                        x-on:click="inputType = 'text'"
+                        x-show="inputType === 'password'"/>
+                    <x-filament::icon-button
+                        icon-alias="forms::components.text-input.hide-password"
+                        icon="heroicon-o-eye-slash"
+                        color="gray"
+                        x-on:click="inputType = 'password'"
+                        x-show="inputType !== 'password'" x-cloak/>
+                </div>
+            @endif
```

Comment:
Could you add a blank line between elements, sort attributes alphabetically and fix formatting issues?

Reply:
Generally, we always add a blank line just between sibling blocks. 🙂

Reply:
Roger that 🫡